docs/technical/conflicts.md

```diff
@@ -57,9 +57,25 @@ rebased to head without resolving conflicts and still not get messy recursive
 conflicts.
 
 As another example, let's go through what happens when you back out a conflicted
```

Comment:
See the discussion in https://github.com/jj-vcs/jj/pull/6415#discussion_r2061055883.